arduino/smartcar/smartcar.ino

```diff
@@ -76,6 +77,7 @@ void loop() {
   if (mqtt.connected()) {
     mqtt.loop();
     //delay to not overload the CPU
-    delay(1);
+    delay(100);
+    mqtt.publish("/smartcar/derivedData/speed", speed);
```

Comment:
Dont think that was what is needed by the app, since you arent publishing "speed" but the throttle percentage. Please fix this @Lancear

Reply:
Just a simple name change as speed comes from the odometer topic

Reply:
Topic was added to list of const topics in later commits on my branch

Reply:
This was just a demo implementation to test the tests :sweat_smile:. The actual implementation is covered by #8.

Reply:
Therefore I would ignore this for now, and when the implementation is added it will be fixed automatically.

Reply:
> Topic was added to list of const topics in later commits on my branch

Also, make sure to update the topics on the wiki and notify everyone when topics change so we are all aware ;)

Reply:
Then I can update the topic names in both the android facade and the arduino test implementation sketch @GeorgZs.